generator/README.md

```diff
@@ -82,14 +82,17 @@ modules:
                             # Required if context is configured on the device.
 
     lookups:  # Optional list of lookups to perform.
-              # This must only be used when the new index is unique.
+              # If keep_source_indexes is the default of false, this must only
```

Comment:
Hrm, this is now more awkward.

```
The default for `keep_source_indexes` is false. Indexes must be unique for this option to be used.
```

I also think we should make the default be `true`, in order to provide safer behavior.

Reply:
@RichiH Do you have any wording suggestions here? Also what do you think about the default value?

Reply:
Changed. The default value should be false, as adding additional labels should only be done on a case-by-case basis - not blindly.